ClassTask/Country.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace ClassTask
+{
+    public abstract class Country : ICountry
+    {
+        public Country(string name, int area, int population, string predominantNationality, string prevailingReligion, 
+            string officialLanguage, string currency, long gdp, string politicalSystem, bool isConquered)
```

Comment:
Too many params for constructor, let's try to simplify this one (add Struct that will contain all values or make some of values optional )